src/Domain/CoordinateMaps/Interval.cpp

```diff
@@ -63,23 +65,26 @@ std::array<tt::remove_cvref_wrap_t<T>, 1> Interval::operator()(
     }
     case Distribution::Logarithmic: {
       const double logarithmic_zero =
-          0.5 * (log((b_ - singularity_pos_) * (a_ - singularity_pos_)));
+          0.5 * (log((b_ - singularity_pos_.value()) *
```

Comment:
It would be quite a bit shorter to define `const double singularity_pos = singularity_pos_.value()` at the beginning. Not asking you to change this unless you get the chance while doing something else.